package/cloudshell/iac/terraform/downloaders/downloader.py

```diff
@@ -9,15 +10,20 @@ def __init__(self, shell_helper: ShellHelperObject):
 
     def download_terraform_module(self) -> str:
         url = self._shell_helper.tf_service.github_terraform_module_url
-        if hasattr(self._shell_helper.tf_service, 'github_token'):
-            token = self._shell_helper.api.DecryptPassword(self._shell_helper.tf_service.github_token).Value
-        else:
-            token = ""
+        attr_handler = ServiceAttrHandler(
+            self._shell_helper.api,
+            self._shell_helper.sandbox_id,
+            self._shell_helper.tf_service
+        )
+
+        token = attr_handler.get_attribute('github_token')
+        branch = attr_handler.get_attribute('branch')
```

Comment:
why not use the attribute name constants?

Reply:
correct...will fix